elements/src/components/pos-login-form/pos-login-form.tsx

```diff
@@ -0,0 +1,49 @@
+import { Component, Event, EventEmitter, h, Prop, State, Watch } from '@stencil/core';
+
+@Component({
+  tag: 'pos-login-form',
+  shadow: {
+    delegatesFocus: true,
+  },
+})
+export class PosLoginForm {
+  @State() idpUrl: string = "http://localhost:3000";
+
+  @State() canSubmit: boolean = true;
+
+  /**
+   * Emits the selected IDP URL to use for login
+   */
+  @Event({ eventName: 'idpUrlSelected' }) idpUrlSelected: EventEmitter;
+  @Event({ eventName: 'pod-os:error' }) errorEmitter: EventEmitter;
+
+  @Watch('idpUrl')
+  validate() {
+    this.canSubmit = Boolean(this.idpUrl);
+  }
+
+  render(): any {
+    return (
+      <form method="dialog" onSubmit={e => this.handleSubmit(e)}>
+        <label htmlFor="idpUrl">URL</label>
+        <input id="idpUrl" type="text" value={this.idpUrl} onInput={e => this.handleChange(e)} />
```

Comment:
a low hanging fruit here could be to add a `<datalist>`with the most common pod providers. Then we will get a typeahead autocomplete, while still being able to enter any provider url or use browser autofill

Reply:
Wow! That was easy. Didn't know it was possible.

Reply:
Suggested issuers added from https://github.com/SolidOS/solid-logic/blob/main/src/issuer/issuerLogic.ts